### PR TITLE
fix issue #103764 Add test case in pkg/cm/cpuset to verify no elements are added after result is called

### LINKED ISSUE #103764: Add test case in pkg/cm/cpuset to verify no elements are added after result is called
This is caught by static check PR
https://github.com/kubernetes/kubernetes/pull/103723/files/7e980a3330b648bcc8952356b16fc7500a8c530f#r670801768

Test case needs to be added for the following code path

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpuset/cpuset.go#L49-#L52

A simple verification of adding element `b.Add()` after result `b.Result()`  is called needs to be added [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpuset/cpuset_test.go#L38
) 
The test should fail until  PR #103723 is merged.

### PR DESCRIPTION
**Pull request by Mangosata**
/kind failing-test

#### What this PR does / why we need it:
caught by this PR https://github.com/kubernetes/kubernetes/pull/103723/files/7e980a3330b648bcc8952356b16fc7500a8c530f#r670801768

add Add() to the test

#### Which issue(s) this PR fixes:

Fixes #103764

#### Special notes for your reviewer:
fixed it following #103764 description, if there is any mistake, plz notify me :)

#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

**Comment by Mangosata** [comment]
signed CLA

File: pkg/kubelet/cm/cpuset/cpuset_test.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ func TestCPUSetBuilder(t *testing.T) {
 	if len(elems) != result.Size() {
 		t.Fatalf("expected cpuset %s to have the same size as %v", result, elems)
 	}
+	b.Add()
```
Comment (ehashman) [inline pkg/kubelet/cm/cpuset/cpuset_test.go:39]: This test seems a bit too coverage-oriented, can we please capture the result and verify the behaviour of add?

**Review by ehashman — CHANGES_REQUESTED** [review]
Shouldn't the test be failing according to https://github.com/kubernetes/kubernetes/issues/103764#issue-947544665 ?

File: pkg/kubelet/cm/cpuset/cpuset_test.go (hunk 1 above)
Comment (Mangosata) [reply pkg/kubelet/cm/cpuset/cpuset_test.go:39]: > This test seems a bit too coverage-oriented, can we please capture the result and verify the behaviour of add?

This test can pass, but it also can pass without b.Add(). I added this according to #103764. If I remove b.Add(), it should fail, but it doesn't.

**Comment by dims** [comment]
Is this PR still needed, please rebase if so (or we can close it?)

**Comment by adisky** [comment]
@Mangosata I believe it is fixed by this PR https://github.com/kubernetes/kubernetes/pull/103415 
closing this, since the issue already has been fixed
please feel to reopen if some tit bits remaining 
/close

**Pull request closed without merging**